src/electron/platform/android/setup/steps/configuring-port-forwarding.ts

```diff
@@ -1,19 +1,36 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+import { AndroidSetupStoreCallbacks } from 'electron/flux/types/android-setup-state-machine-types';
+import { AndroidSetupDeps } from 'electron/platform/android/setup/android-setup-deps';
 import { AndroidSetupStepConfig } from 'electron/platform/android/setup/android-setup-steps-configs';
 
-export const configuringPortForwarding: AndroidSetupStepConfig = (stepTransition, deps, store) => ({
+const removeOldForwardedPort = async (
+    deps: AndroidSetupDeps,
+    store: AndroidSetupStoreCallbacks,
+) => {
+    const existingPort = store.getScanPort();
+    if (existingPort) {
+        try {
+            deps.logger.log(`removing old tcp:${existingPort} forwarding`);
+            await deps.removeTcpForwarding(existingPort);
+            store.setScanPort(null);
```

Comment:
Don't we want to set the scan port to null regardless of whether or not an exception was thrown?

Reply:
Yup. Moved after the try/catch block